{"kind":"symbol","schemaVersion":{"patch":0,"major":0,"minor":3},"sections":[],"identifier":{"url":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/errorDescription","interfaceLanguage":"swift"},"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"? { ","kind":"text"},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}],"languages":["swift"],"platforms":["iOS"]}],"kind":"declarations"}],"hierarchy":{"paths":[["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/LocalizedError-Implementations"]]},"metadata":{"symbolKind":"property","modules":[{"name":"VigramSDK"}],"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"externalID":"s:9VigramSDK19AuthenticationErrorO16errorDescriptionSSSgvp","extendedModule":"VigramSDK","roleHeading":"Instance Property","role":"symbol","title":"errorDescription"},"abstract":[{"text":"Inherited from ","type":"text"},{"type":"codeVoice","code":"LocalizedError.errorDescription"},{"type":"text","text":"."}],"variants":[{"paths":["\/documentation\/vigramsdk\/authenticationerror\/errordescription"],"traits":[{"interfaceLanguage":"swift"}]}],"references":{"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/LocalizedError-Implementations":{"abstract":[],"role":"collectionGroup","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/LocalizedError-Implementations","kind":"article","title":"LocalizedError Implementations","type":"topic","url":"\/documentation\/vigramsdk\/authenticationerror\/localizederror-implementations"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/errorDescription":{"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/errorDescription","kind":"symbol","url":"\/documentation\/vigramsdk\/authenticationerror\/errordescription","abstract":[],"fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"type":"topic","role":"symbol","title":"errorDescription"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError":{"abstract":[{"type":"text","text":"AuthenticationError contains custom failure events when connecting to Authentication process."}],"role":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError","kind":"symbol","title":"AuthenticationError","type":"topic","url":"\/documentation\/vigramsdk\/authenticationerror","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"AuthenticationError","kind":"identifier"}],"navigatorTitle":[{"kind":"identifier","text":"AuthenticationError"}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK":{"type":"topic","role":"collection","title":"VigramSDK","images":[{"type":"icon","identifier":"logo.png"}],"url":"\/documentation\/vigramsdk","abstract":[],"kind":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK"},"logo.png":{"alt":"Vigram logo.","variants":[{"traits":["1x","light"],"url":"\/images\/com.vigram.VigramSDK\/logo.png"}],"type":"image","identifier":"logo.png"}}}